package.json

```diff
@@ -21,19 +21,18 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^8.8.0",
+    "@theforeman/vendor": "^8.15.0",
     "eslint-plugin-spellcheck": "0.0.17",
     "intl": "~1.2.5",
-    "jed": "^1.1.1",
-    "react-intl": "^2.8.0"
+    "jed": "^1.1.1"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": ">= 8.7, < 8.12.1",
-    "@theforeman/eslint-plugin-foreman": ">= 8.7, < 8.12.1",
-    "@theforeman/stories": ">= 8.7, < 8.12.1",
-    "@theforeman/test": ">= 8.7, < 8.12.1",
-    "@theforeman/vendor-dev": ">= 8.7, < 8.12.1",
+    "@theforeman/builder": ">= 8.7, < 8.15.0",
```

Comment:
I think I'd keep these simple and also make them `^8.15.0`.

Reply:
`^8.15.0` is not the same as `>= 8.7, < 8.15.0`, but given we bump `vendor` to `^8.15.0` using the same here sounds reasonable.

But the < 8.12.1 pin was there for a reason